src/statics/voidAll.test.js

```diff
@@ -47,15 +45,14 @@ describe('voidAll', () => {
     ).rejects.toThrow(new Error('dead'));
   });
 
-  it('should reject iterable errors', async () => {
-    await expect(
+  it('should reject iterable errors', () =>
+    expect(
```

Comment:
```suggestion
    return expect(
```

Reply:
It's a bare `() =>` - not a block function